packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/print-auth-exists-warning.ts

```diff
@@ -18,5 +18,5 @@ export const printAuthExistsWarning = (context: $TSContext): void => {
 };
 
 const checkAuthIsImported = (authResources: [string, $TSObject][]): boolean => {
-  return authResources.filter(([_, resource]) => resource?.serviceType === 'imported').length > 0;
+  return authResources.filter(([, resource]) => resource?.serviceType === 'imported').length > 0;
```

Comment:
I was looking at eliminating unused `_` the other day. I didn't know this is possible. Thank you.